
string[1] string_array